src/components/HWalletConnection.jsx

```diff
@@ -19,11 +19,7 @@ const steps = {
     },
     {
       "icon": <ApplicationSettingsIcon/>,
-      "text": "Ensure the Browser Support is enabled in Settings"
```

Comment:
Are you sure that you can combine those? Better show the UI diff to Thomas so he can decide. He is quite picky about those things.